modules/__tests__/Media-server-test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment node
+ */
+
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import Media from '../Media'
+
+describe('A <Media> in server environment', () => {
+  const queries = {
+    sm: '(max-width: 1000px)',
+    lg: '(max-width: 2000px)',
+    xl: '(max-width: 3000px)',
+  }
+
+  describe('when no default matches prop provided', () => {
+    const expected = {
+      sm: true,
+      lg: true,
+      xl: true,
+    }
+
+    it('should call children function with all queries matching', () => {
+      ReactDOMServer.renderToStaticMarkup(
+        <Media queries={queries}>
+          {matches => {
+            expect(matches).toMatchObject(expected)
```

Comment:
Let's make the assertion on the markup instead of in the render prop and change the assertion to be something like "it renders all queries".